### PR TITLE
abi-bits: add domainname to utsname in mlibc abi

### PR DESCRIPTION
**Pull request by 64**
As per the discussion in #504.

### CODE REVIEW

File: ABI_BREAKS.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ This document lists the ABI breaks that were made in each mlibc major version.
 
 - [#452](https://github.com/managarm/mlibc/pull/452): The functions `FD_{CLR,ISSET,SET,ZERO}` were renamed to `__FD_{CLR,ISSET,SET,ZERO}` and replaced by macros to match Wine's assumptions.
 - [#511](https://github.com/managarm/mlibc/pull/511): Musl's regex engine was added, implementing `regcomp` and `regexec`. This required some changes to the `regex_t` struct.
-- [#504](https://github.com/managarm/mlibc/pull/504): A `domainname` member was added to `struct utsname`, which is a glibc extension.
+- [#580](https://github.com/managarm/mlibc/pull/504): A `domainname` member was added to `struct utsname`, which is a glibc extension.
```
Comment (Geertiebear) [inline ABI_BREAKS.md:9]: But the old entry is still true? We should just add two entries here.

**Review by Geertiebear — CHANGES_REQUESTED** [review]

File: ABI_BREAKS.md (hunk 1 above)
Comment (64) [reply ABI_BREAKS.md:9]: The old entry changed the Linux ABI, which isn't really something that downstream users will care about

Comment (Geertiebear) [reply ABI_BREAKS.md:9]: There should still be two entries. We cannot assume that no user is interested in the changelog of linux ABI changes.

Comment (64) [reply ABI_BREAKS.md]: Fair point.

**Pull request merged**